docs/getting-started/installing-and-running.mdx

```diff
@@ -0,0 +1,144 @@
+       .:::::‘‘‘‘:::::.
+      .::::        ::::.
+ .::::::::          ::::::::.
+ `::::::::          ::::::::‘
+      `::::        ::::‘
+       `:::::....:::::‘
+         `::::::::::‘        Conduit v0.10.0 linux/amd64
+             ‘‘‘‘
+2024-02-20T21:37:45+00:00 INF All 0 tables opened in 0s component=badger.DB
+2024-02-20T21:37:45+00:00 INF Discard stats nextEmptySlot: 0 component=badger.DB
+2024-02-20T21:37:45+00:00 INF Set nextTxnTs to 0 component=badger.DB
+2024-02-20T21:37:45+00:00 INF builtin plugins initialized component=builtin.Registry count=6
+2024-02-20T21:37:45+00:00 WRN could not read plugin directory error="open /app/connectors: no such file or directory" component=standalone.Registry
+2024-02-20T21:37:45+00:00 INF standalone plugins initialized component=standalone.Registry count=0 plugin_path=/app/connectors
+2024-02-20T21:37:45+00:00 INF processors initialized component=processor.Service count=0
+2024-02-20T21:37:45+00:00 INF connectors initialized component=connector.Service count=0
+2024-02-20T21:37:45+00:00 INF pipelines initialized component=pipeline.Service count=0
+2024-02-20T21:37:45+00:00 INF pipeline configs provisioned component=provisioning.Service created=[] deleted=[] pipelines_path=./pipelines
+2024-02-20T21:37:45+00:00 INF grpc server started address=[::]:8084
+2024-02-20T21:37:45+00:00 INF http server started address=[::]:8080
+2024-02-20T21:37:45+00:00 INF
+2024-02-20T21:37:45+00:00 INF click here to navigate to Conduit UI: http://localhost:8080/ui
+2024-02-20T21:37:45+00:00 INF click here to navigate to explore the HTTP API: http://localhost:8080/openapi
+2024-02-20T21:37:45+00:00 INF
+```
+
+You should now be able to interact with the Conduit UI and HTTP API on port 8080.
+  </TabItem>
+  <TabItem value="source" label="Building from source">
+Before you can build Conduit from source, you need to have Go 1.21 or higher installed. The Go website has [comprehensive instructions](https://go.dev/doc/install) on how to get the language installed on your machine. Once installed, you can follow the rest of the instructions.
+
+1. Start by downloading the source code from the latest stable release on the Conduit [Releases Page](https://github.com/ConduitIO/conduit/releases/latest). Alternatively, you can run this command to automatically download the latest stable source to your current directory:
+
+```shell
+$ TAG=$(curl -s https://api.github.com/repos/ConduitIO/conduit/releases/latest | grep "tag_name\": \"v[0-9.]*" | awk -F \" '/tag_name\": \"*\"/{print $(NF-1)}') && curl -o conduit.tgz -L https://github.com/ConduitIO/conduit/tarball/$TAG
+```
+
+A file called `conduit.tgz` will be in your current directory. The next step is to expand the source:
+
+```shell
+$ tar zxvf conduit.tgz
+```
+
+Then change directories to the appropriate folder. Keep in mind that the folder name might be different between releases since it's tied to the latest git sha for the commit.
+
+```shell
+$ cd ConduitIO-
```

Comment:
This looks a bit strange, I would specify the whole folder name, even though it will change in future releases. I think people will get what we mean.
```suggestion
$ cd ConduitIO-conduit-0ce62ec
```